Mathlib/CategoryTheory/Limits/FunctorCategory.lean

```diff
@@ -445,3 +447,221 @@ def colimitIsoSwapCompColim [HasColimitsOfShape J C] (G : J ⥤ K ⥤ C) :
 #align category_theory.limits.colimit_iso_swap_comp_colim CategoryTheory.Limits.colimitIsoSwapCompColim
 
 end
+
+section prod
+
+/-- `functorProd F G` is the explicit binary product of type-valued functors `F` and `G`. -/
+def functorProd (F G : C ⥤ Type w) : C ⥤ Type w where
```

Comment:
As the linter complains, I would suggest moving everything to a new file `Limits.Shapes.FunctorToTypes`, and renaming this definition as `CategoryTheory.FunctorToTypes.prod`.